arangod/IResearch/IResearchDataStore.h

```diff
@@ -505,6 +508,8 @@ class IResearchDataStore {
   std::mutex _commitMutex;
 
   // for insert(...)/remove(...)
+  irs::IndexWriter::Transaction _recoveryTrx;
```

Comment:
at least this transaction should be aborted explicitly in postRecoveryCallback. But iy would be better to not create it at all if not in recovery.

Reply:
Fixed